.travis.yml

```diff
@@ -0,0 +1,23 @@
+branches: {only: [master, develop]}
+sudo: false
+language: python
+python: 3.7
+cache: pip
+
+addons:
+  postgresql: 12
+
+matrix:
+  include:
+    - env: TEST_PART=tests DATABASE_URL=postgres:///kukkuu SECRET_KEY=topsecret123
```

Comment:
travis now supports giving names to certain jobs in the matrix, which gives you nicer syntax than `TEST_PART` env var.
```yml
- name: Unit tests
   env: DATABASE_URL=postgres:///kukkuu SECRET_KEY=topsecret123
   services: [postgresql]
...
```

Reply:
Ok cool, changing to that.